src/components/flowchart/flowchart.js

```diff
@@ -448,6 +449,44 @@ export class FlowChart extends Component {
     node && this.showTooltip(event, node.fullName);
   };
 
+  /**
+   * Enable a layer's active state when hovering it, update labelName's active className accordingly
+   * @param {Object} event Event object
+   * @param {Object} node Datum for a single node
+   */
+  handleLayerMouseOver = (event, node) => {
```

Comment:
`event` isn't being used here, though when I remove it locally the highlight doesn't show up anymore. Do you see the same? I'd think you could take it out as an argument, and in the `mouseOut` function as well.

Reply:
hmm i don't think it will work, as the `event` is always the first parameter of the `mouseEvent` callback, right? And `node` is a data we passed it to this function. I think if we remove `event` and it will think `node` is as `event`. I guess that's why the highlight wouldn't work

Reply:
Makes sense!